lib/behavior.js

```diff
@@ -0,0 +1,170 @@
+'use strict';
+
+/**
+ * Built-in behaviors.
+ * These provide hooks into modifying Mocha's behavior for different use cases.
+ */
+
+var utils = require('./utils');
+
+/**
+ * Default Mocha behavior
+ */
+exports.Default = {
```

Comment:
Can we rename this from `Default` to something else?
- It is a little confusing with es6 default modules.
- Whenever we want to change the default behavior to another one, we have to refactor 2 places now.
- A semantic name indicating what it does is more meaningful IMO. `noCtxAsArg` or something in this context (pun intended 😄 ).

Reply:
yeah, probably a good idea to rename this, or just use `module.exports` and put the `Functional` behavior within the `func` interface... what do you think of that?